siliconcompiler/scheduler/send_messages.py

```
@@ -219,24 +219,24 @@ def send(chip, msg_type, step, index):
             do_send = True
         except smtplib.SMTPAuthenticationError as e:
-            chip.logger.error(f'Unable to authenticate to email server: {e}')
+            project.logger.error(f'Unable to authenticate to email server: {e}')
         except Exception as e:
-            chip.logger.error(f'An error occurred during login to email server: {e}')
+            project.logger.error(f'An error occurred during login to email server: {e}')
 
         if do_send:
             try:
                 smtp_server.sendmail(msg['From'], to, msg.as_string())
             except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)